### PR TITLE
Display raw UINT/INT values as hex

### LINKED ISSUE #775: Get Details dialog should show RAW as hex
The Get Details dialog available by right clicking a telemetry item should show numeric RAW values as hex.

### PR DESCRIPTION
**Pull request by jmthomas**
closes #775

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue [hunk 1]
```diff
@@ -292,7 +292,14 @@ export default {
...
                 }
-                this.rawValue = values[0][0]
+                if (
+                  this.details.data_type.includes('INT') &&
+                  !this.details.array_size
+                ) {
+                  this.rawValue = '0x' + values[0][0].toString(16).toUpperCase()
+                } else {
+                  this.rawValue = values[0][0]
+                }
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue:302]: I wanted hex to be in addition to the decimal display, not replace it.

**Review by ryanmelt — CHANGES_REQUESTED** [review]

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue (hunk 1 above)
Comment (jmthomas) [reply openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue:302]: Something like `0xA (10)`

Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue:302]: That or a separate line.  Either would be fine.

Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue:302]: Of those: 10 (0xA)

**Comment by jmthomas** [comment]
Screenshot 2023-09-12 at 10 41 55 PM

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**